requirements.txt

```diff
@@ -1,38 +1,185 @@
 absl-py==0.9.0
+alembic==1.3.2
+amqp==2.5.2
+appdirs==1.4.3
+appnope==0.1.0
+APScheduler==3.6.3
+arrow==0.15.5
+aspy.yaml==1.3.0
 astor==0.8.1
+astroid==2.3.3
+astunparse==1.6.3
+attrs==19.3.0
+backcall==0.1.0
+billiard==3.6.1.0
+black==19.10b0
+bleach==3.1.0
+blinker==1.4
+boto==2.49.0
 cachetools==4.0.0
+celery==4.4.0
 certifi==2019.11.28
+cffi==1.13.2
+cfgv==2.0.1
 chardet==3.0.4
-gast==0.2.2
+click==7.1.2
+coverage==4.5.4
+cryptography==2.8
+decorator==4.4.2
+des==1.0.5
+diff-match-patch==20181111
+distro==1.4.0
+dnspython==1.16.0
+dredd-hooks==0.2.0
+elasticsearch==7.1.0
+elasticsearch-dsl==7.0.0
+envparse==0.2.0
+eventlet==0.25.1
+factory-boy==2.12.0
+Faker==3.0.0
+Flask==1.1.2
+Flask-Admin==1.5.4
+Flask-Caching==1.8.0
+Flask-CeleryExt==0.3.2
+Flask-Cors==3.0.8
+Flask-Elasticsearch==0.2.5
+Flask-JWT-Extended==3.24.1
+Flask-Limiter==1.1.0
+Flask-Login==0.4.1
+Flask-Migrate==2.5.2
+flask-redis==0.4.0
+Flask-REST-JSONAPI==0.12.6
+Flask-Script==2.0.6
+Flask-Scrypt==0.1.3.6
+Flask-SQLAlchemy==2.4.1
+Flask-WTF==0.14.3
+forex-python==1.5
+gast==0.3.3
+geoip2==3.0.0
 google-auth==1.11.3
 google-auth-oauthlib==0.4.1
+google-compute-engine==2.8.13
 google-pasta==0.2.0
+greenlet==0.4.15
 grpcio==1.27.2
+gunicorn==20.0.4
 h5py==2.10.0
+healthcheck==1.3.3
+html5lib==1.0.1
+httplib2==0.15.0
+humanize==0.5.1
+icalendar==4.0.4
+identify==1.4.11
 idna==2.9
+importlib-metadata==1.3.0
+ipykernel==5.2.0
+ipython==7.13.0
+ipython-genutils==0.2.0
+isort==4.3.21
+itsdangerous==1.1.0
+jedi==0.16.0
+Jinja2==2.11.2
 joblib==0.14.1
-Keras==2.3.1
+jupyter-client==6.1.2
+jupyter-core==4.6.3
+Keras==2.4.3
 Keras-Applications==1.0.8
-Keras-Preprocessing==1.1.0
+Keras-Preprocessing==1.1.2
+kombu==4.6.7
+lazy-object-proxy==1.4.3
+limits==1.4.1
+Mako==1.1.0
 Markdown==3.2.1
+MarkupSafe==1.1.1
+marrow.mailer==4.0.3
+marrow.util==1.2.3
+marshmallow==2.15.6
+marshmallow-jsonapi==0.22.0
+maxminddb==1.5.2
+mccabe==0.6.1
+monotonic==1.5
+more-itertools==8.0.2
+nodeenv==1.3.4
+nose==1.3.7
 numpy==1.18.2
+oauth2==1.9.0.post1
 oauthlib==3.1.0
+omise==0.8.1
 opt-einsum==3.2.0
+parso==0.6.2
+pathspec==0.7.0
+paypalrestsdk==1.13.1
+pep8==1.7.1
+pexpect==4.8.0
+pickleshare==0.7.5
 Pillow==7.0.0
+pre-commit==1.21.0
+prompt-toolkit==3.0.5
 protobuf==3.11.3
+psycopg2-binary==2.8.4
+ptyprocess==0.6.0
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
+pycairo==1.19.0
+pycountry==19.8.18
+pycparser==2.19
+pycryptodome==3.9.4
+Pygments==2.6.1
+PyGObject==3.34.0
+PyJWT==1.7.1
+pylint==2.4.4
+pyOpenSSL==19.1.0
+PyPDF2==1.26.0
+PyQt5-sip==4.19.20
```

Comment:
It’s very surprising, `4.19.20` is not even listed from <https://pypi.org/project/PyQt5-sip/#history>.

Please remove unneeded dependencies.

Reply:
Thanks